README.md

```diff
@@ -114,7 +118,7 @@ After the server is running, modify the `.env` file inside the repository's root
 export $((echo "MLFLOW_TRACKING_URI=http://127.0.0.1:5000" >> .env; cat .env) | xargs)
 ```
 
-## Training The Model
+## Training The Model {#training-the-model}
```

Comment:
You shouldn't need the {#training-the-model} portion of the TOC to work correctly in GitHub or Visual Studio Code.

Reply:
Agreed.